### ISSUE TITLE
Rename in Scope menu option in Desktop IDE is missing letter underlined keyboard shortcut.

### ISSUE THREAD

**Issue by R-Hannibal** [rstudio/rstudio#6528]
### System details

    RStudio Edition : Desktop
    RStudio Version : Version 1.3.926
    OS Version      : Windows 10

### Steps to reproduce the problem

1. Open RStudio desktop on a Windows or Linux OS.
2. Press the Alt key to activate underlined keyboard shortcut functionality.
3. Select Code.

### Describe the problem in detail
The Rename in Scope menu option is missing an underlined keyboard shortcut.
This issue also occurs on Linux OS, tested by @dfalty.
Similar to issue #5610. 
![Rename in Scope](https://user-images.githubusercontent.com/62304541/77585659-d8473200-6eb2-11ea-8a26-311e2e02f75b.png)

### Describe the behavior you expected
All menu options include a underlined keyboard shortcut.

- [x ] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x ] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
